tldw_Server_API/tests/Jobs/test_jobs_repository.py

```
@@ -0,0 +1,90 @@
+from __future__ import annotations
+
+from datetime import datetime, timezone
+
+import pytest
+
+from tldw_Server_API.app.core.Jobs.migrations import ensure_jobs_tables
+
+
+@pytest.fixture()
```

Ruff (0.15.6)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)